host/class/uac/usb_host_uac/test_app/pytest_usb_host_uac.py

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Tuple
+
+import pytest
+from pytest_embedded_idf.dut import IdfDut
+
+
+@pytest.mark.esp32s2
+@pytest.mark.esp32s3
+@pytest.mark.temp_skip_ci(targets=['esp32s2', 'esp32s3'], reason='unable to mock UAC 1.0 device with tinyusb')
+@pytest.mark.usb_host
```

Comment:
Hopefully the last CI fix:
Please remove the `@pytest.mark.temp_skip_ci` as it only work in the esp-idf CI. 

As an alternative, we can delete the `@pytest.mark.usb_host`, to prevent pytest from running this test case. We can add the proper runner marker, once the CI is extended with the new runners tesing this UAC class.

The pytest called from `build_and_run_test_app_usb.yml` looks only for `usb_host` markers. So it should not collect this test case at all, since it does not have the `usb_host` marker.

```suggestion
# No runner marker, unable to mock UAC 1.0 device with tinyusb
@pytest.mark.esp32s2
@pytest.mark.esp32s3
```

Reply:
Thanks @peter-marcisovsky